jme3-desktop/src/main/java/jme3tools/converters/ImageToAwt.java

```diff
@@ -104,13 +104,21 @@ public DecodeParams(int bpp, int rm, int rs, int im, int is){
         final int mxxxx = 0xffffffff;
         final int sxxxx = 0;
 
+        @SuppressWarnings("unused")

```

Comment:
A comment (just in front of the first one) about why these really are used would be good for future understanding